cmd/makedocs/check-docs_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+var docCheckers []checkerDoc
+
+func parseFiles() []checkerDoc {
+	tests, err := ioutil.ReadDir("./testdata")
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	pkgs, err := parser.ParseDir(&token.FileSet{}, "./testdata",
+		func(inf os.FileInfo) bool {
```

Comment:
Can be be extracted to a variable `filter`, will be a bit simpler to read the code.